package.json

```diff
@@ -51,6 +52,7 @@
     "json-loader": "0.5.4",
     "load-grunt-configs": "1.0.0",
     "load-grunt-tasks": "3.5.2",
+    "lodash": "^4.17.4",
```

Comment:
Do you need all of **lodash**, or just [**lodash.ismatchwith**](https://www.npmjs.com/package/lodash.ismatchwith)?